config/tools-manual.json

```diff
@@ -81,12 +81,12 @@
                 "title": "Springwolf",
                 "description": "Automated documentation for async APIs built with Spring Boot. Like Springfox for AsyncAPI. Auto-generates an AsyncAPI document and a web UI.",
                 "links": {
-                    "websiteUrl": "https://springwolf.github.io/",
+                    "websiteUrl": "https://www.springwolf.dev",
                     "repoUrl": "https://github.com/springwolf/springwolf-core"
                 },
                 "filters": {
                     "language": "Java",
-                    "technology": ["Springboot"],
+                    "technology": ["Spring Boot", "Gradle"],
```

Comment:
for consistency, can you please update all entries of `Springboot` in this file?